config/locales/en.yml

```diff
@@ -124,12 +124,10 @@ en:
       last_name: Last name
       ssn: Social Security Number
       dob: Date of birth
-      ccn: Credit Card Number
+      ccn: Last 8 Digits of your Credit Card Number
```

Comment:
Is ccn ca common enough acronym to use here?  I didn't know what it was right away.. but i've never worked on on a system that delt with them so i might not be a good data point.  What not 'credit_card_number'?